tests/backends/base/test_base.py

```diff
@@ -211,6 +212,21 @@ def test_wrapper_connection_specific(self):
         self.assertEqual(connection.execute_wrappers, [])
         self.assertEqual(connections["other"].execute_wrappers, [])
 
+    @skipIf(connection.vendor == "sqlite", "can't see cursor's executed queries")
+    @skipIf(
+        connection.settings_dict.get("OPTIONS").get("server_side_binding") is True,
+        "no debug info for psycopg server-side binding",
+    )
```

Comment:
We should add this test to the `django_test_expected_failures` in PostgreSQL's `DatabaseFeatures` instead of using `@skipIf`.